### PR TITLE
User(landlord) can create a listing

### PR DESCRIPTION
**Pull request by Alex-Bellamy**
### User Story
```
As a landlord
In order to post my space for bidding
I would like to be able to complete a form
```
## Changes proposed in this pull request:
* Login flow
* Create listing form
## Packages added
- JTockAuth
## Screenshots (Client)
create

### CODE REVIEW

File: cypress/integration/userCanCreateListings.feature.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      cy.get("[data-cy=email]").type("user@mail.com");
+      cy.get("[data-cy=password]").type("password");
+      cy.get("[data-cy=button]").contains("Submit").click();
+    });
+  });
+
+  it("user can click on create listing", () => {
+    cy.get("[data-cy=button]").contains("Rent out a space!").click();
+    cy.get("[data-cy=listing-form]").within(() => {
+      cy.get("div[role='option']").contains("Parking").click({force: true});
```
Comment (Odeane) [inline cypress/integration/userCanCreateListings.feature.js]: Why did you need to force the button to click??

File: src/components/CreateListing.jsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+  const categoryOption = [{ key: "p", text: "Parking", value: "parking" }];
+
+  const sceneOptions = [
+    { key: "i", text: "Indoors", value: "indoor" },
+    { key: "o", text: "Outdoors", value: "outdoor" },
+  ];
+
+  const submitListing = async (event) => {
+    debugger;
```
Comment (Felix-Willborg) [inline src/components/CreateListing.jsx]: Come on guys!!! lose the debugger!

File: src/components/CreateListing.jsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
+import React, { useState } from "react";
```
Comment (Odeane) [inline src/components/CreateListing.jsx:1]: I thought we were supposed to be using redux for state management

File: src/components/CreateListing.jsx (hunk 2 above)
Comment (Odeane) [inline src/components/CreateListing.jsx]: There is a debugger in your code

**Review by Odeane — APPROVED** [review]

File: cypress/integration/userCanCreateListings.feature.js (hunk 1 above)
Comment (Alex-Bellamy) [reply cypress/integration/userCanCreateListings.feature.js]: styling is creating a conflict, we will be dealing with the styling over the weekend and will remove the force command then

**Review by Odeane — COMMENTED** [review]
I saw no sad paths

**Review by oliverochman — APPROVED** [review]
